src/FixedDataTableCell.js

```diff
@@ -193,57 +192,39 @@ class FixedDataTableCell extends React.Component {
       isHeaderOrFooter,
       visible,
       zIndex,
-      left1,
-      // position,
+      scrollOffsetLeft,
       ...props
     } = this.props;
-    // debugger
-    // var style = {
-    //   height,
-    //   position:'absolute',
-    //   width,
-    //   zIndex,
-    //   visibility: visible ? 'visible' : 'hidden',
-    // };
-
-    // console.log(this._initialRender,"Hello")
-    // if (this.props.isRTL) {
-    //   style.right = props.left;
-    // } else {
-    //   style.left = props.left;
-    // }
-    // FixedDataTableTranslateDOMPosition(
-    //   style,
-    //   -1*(left1),
-    //   0,
-    //   this.props.initialRender,
-    //   this.props.isRTL
-    // );
-
-    // if (this.props.isRTL) {
-    //   style.right =this.props.left+this.props.offsetLeft-left1;
-    // } else {
-    //   style.left = this.props.left+this.props.offsetLeft-left1;
-    // }
+
     var style = {
       height,
+      position: 'absolute',
       width,
+      zIndex,
       visibility: visible ? 'visible' : 'hidden',
     };
 
```

Comment:
The variable which differentiate the different types of Cell groups (fixed,scrollable,fixed-right) is zIndex .It will be set to 2 for fixed and set to 0 for scrollable .Earlier we were providing styling to the whole cell group so we were using zIndex in FixedDataTableCellGroup .But now to maintain those styles we have passed zIndex to each and every cell to provide styles to it .